src/app/components/login/login.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container w-25">
+  <div class="row justify-content-center mt-5">
+    <h2 class="text-center mb-4">Bienvenido:</h2>
+    <form>
+      <div class="form-group">
+        <label for="usuario">Usuario:</label>
+        <input
+          type="text"
+          class="form-control"
+          id="usuario"
+          placeholder="Ingresa tu usuario"
+        />
+      </div>
+      <div class="form-group mt-3">
+        <label for="contrasena">Contraseña:</label>
```

Comment:
```suggestion
        <label>Contraseña:</label>
```
why u use the for attribute?